rules/sources/personal_identification.yaml

```diff
@@ -38,14 +38,16 @@ sources:
       - "(?i).*((?:last|sur(?!geon))[^\\s/(;)#|,=!>]{0,5}name)"
     tags:
       law: GDPR
-  
+
   - id: Data.Sensitive.PersonalIdentification.Age
     name: Age
     category: Personal Identification
     isSensitive: False
     sensitivity: low
     patterns:
-      - "(?i)age|is[_-]{0,1}(minor|major|adult|child)|(.*(age[^\\s/(;)#|,=!>]{0,10}(?:years|yrs)|(?:user|person|customer)[^\\s/(;)#|,=!>]{0,5}(?<!im|p|langu|us|aver)age))"
+      - "(?i)^(user|person|customer|under|years|yrs|human)?(age)"
```

Comment:
@gaurav-gogia You may want to relook at this regex as the following will not be detected 
- user_age

Following wrong detections
- ageOfYears
- isLegal

Would recommend a thorough regex testing and comparison with earlier regex as well